LibrarySystem/Queries/8.OverdueAnalysis.sql

```diff
@@ -0,0 +1,17 @@
+SELECT 
+	BK.BookID,
+	BK.Title,
+	B.BorrowerID,
+	B.FirstName,
+	DATEDIFF(DAY, L.DueDate,  GETDATE()) AS DaysOverDue
+FROM 
+	Loans L
+JOIN
+	Books BK ON BK.BookID = L.BookID
+JOIN
+	Borrowers B ON B.BorrowerID = L.BorrowerID
+WHERE
+	DATEDIFF(DAY, L.DueDate,  GETDATE()) > 30
```

Comment:
'30'
it's better to declare such constants as Variables with a meaning full name.